src/content/cloud/registry.mdx

```diff
@@ -50,3 +50,25 @@ Any image pushed into the Okteto Registry is private. You'll need to authenticat
 Namespaces in Okteto Cloud are automatically allowed to pull images that belong to their namespace automatically. If your application uses images from the Okteto Registry, it'll be able to pull container images without any extra configuration.
 
 To pull an image from a different namespace, you'll need to create and configure the required `imagePullSecrets` as outlined [here](/docs/reference/faqs/#how-to-use-private-images).
+
+## Push helm chart into the Okteto Registry
+
+Just as we can push images to the Okteto Registry, you also have the option to push Helm charts. To do this, the first step is to log in to the registry where we want to push the chart:
+
+```
+helm registry login registry.cloud.okteto.net -u my-username -p my-password
```

Comment:
We should explain how to get the user, password and registry url. 
They are set as envvars in the deploy command:
- `${OKTETO_USERNAME}` 
- `${OKTETO_TOKEN}` (password)
- `${OKTETO_REGISTRY_URL}`

We should also encourage the user to use this inside the deploy command, so they have the envvars already set

Reply:
good point. In my opinion talking about those environment variables as such is not necessary in this section but we could suggest the user to check his credentials by using the `okteto context show` command as you suggest.

Reply:
I was thinking that a common use case would be to package and push helm charts on the deploy section of the okteto manifest. In that case it would be easy to add the command `helm registry login ${OKTETO_REGISTRY_URL} -u ${OKTETO_USERNAME} -p ${OKTETO_TOKEN}`

Reply:
so, do you think some kind of *note* could be useful to remind users that their credentials can be found via OKTETO_REGISTRY_URL, OKTETO_USERNAME, OKTETO_TOKEN envs or by running the `okteto context show` i.e. add more info in the last note?

Reply:
I was thinking some kind of note like 

#### In your deploy section, you can login with helm to the okteto registry by adding this command
```
helm registry login ${OKTETO_REGISTRY_URL} -u ${OKTETO_USERNAME} -p ${OKTETO_TOKEN}
```

Reply:
I think it's being too specific, I mean, the user might want to login without using the manifest

Reply:
Yeah, that's true.